docs/operating-scylla/nodetool-commands/getsstables.rst

```diff
@@ -0,0 +1,41 @@
+Nodetool getsstables
+====================
+
+**getsstables** ``[-hf | --hex-format] <keyspace> <table> <key>`` - Get the sstables that contain the given key.
+
+Syntax
+------
+
+.. code-block:: console
+
+   nodetool getsstables [-hf | --hex-format] <keyspace> <table> <key>
+
+Options
+-------
+
+====================================================================  ==================================================================================================================
+Parameter                                                             Description
+====================================================================  ==================================================================================================================
+-hf / --hex-format                                                    Specify the key in hexadecimal string format
```

Comment:
I think (based on the quick look at the code) that the ScyllaDB API endpoint doesn't support this option. I implemented it in the client side, because the Java nodetool also implements it. We can fix the API in a follow-up (it is just a few lines of code -- tools/scylla-types.cc has examples on how to parse hex representations).

Reply:
okay. so it's okay to keep this option in the document, right?